docs/download/repositories.md

```diff
@@ -17,14 +17,14 @@ Academic research cloud infrastructure built to house ICGC data.
 | Property         | Value                                                                                                                                            |
 | :--------------- | ------------------------------------------------------------------------------------------------------------------------------------------------ |
 | Name             | Cancer Genome Collaboratory                                                                                                                      |
-| Contact          | <mailto:dcc-support@icgc.org>                                                                                                      |
+| Contact          | [ICGC ARGO Contact page](https://platform.icgc-argo.org/contact)                                                                                 |
 | Repository type  | Cloud                                                                                                      |
 | ICGC Portal Page | [Portal](https://dcc.icgc.org/repositories?filters=%7B%22file%22:%7B%22repoName%22:%7B%22is%22:%5B%22Collaboratory%20-%20Toronto%22%5D%7D%7D%7D) |
 | Download Client  | [Tarball](/software/download/#score-client), [Docker](https://hub.docker.com/r/overture/score/) |
 | Repo Code        | `collaboratory`                                                                                                                                  |
 
 ### Obtaining Data Access
-Follow the procedure outlined at the [DACO page](https://icgc.org/daco).
+Follow the procedure outlined at the [DACO page][1].
```

Comment:
@hknahal can you change this sentence to: 

Can you change this sentence to:

To obtain access to ICGC Controlled data, learn more about the [DACO application process,](https://docs.icgc-argo.org/docs/data-access/daco/applying)or [start a DACO application here](https://daco.icgc-argo.org/).